.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "src/main/idls"]
 	path = src/main/idls
-	url = git@github.com:uber/cadence-idl.git
```

Comment:
did you mean to change this? This should be `git@` as far as I'm aware

Reply:
Is that a java thing? Why should it be git@? All the other ones are using this format and it seems to be working.
This also seems to be working to me.

Reply:
it refs to the git cloning option, by default git uses SSH as a transport protocol, but you can use HTTP as well, though it's a bit less well supported. 
<img width="356" alt="Screenshot 2025-05-05 at 7 54 13 PM" src="https://github.com/user-attachments/assets/607fecd5-c0ea-4274-9b41-e091d88a0c18" />